ceph.spec.in

```diff
@@ -24,6 +24,7 @@
 %bcond_with zbd
 %bcond_with cmake_verbose_logging
 %bcond_without ceph_test_package
+%bcond_without mgr_dashboard_frontend_support_multi_language
```

Comment:
We should create a separate condition just for building the front-end and then another one for the multi-language (which IMHO should be by default true, not the other way around).

Reply:
Good ideas,I will use `mgr_dashboard_frontend_build` building the front-end and then another one `mgr_dashboard_frontend_support_multi_language` for the multi-language.